### DISCUSSION TITLE
Is there a list of known failure points in lfric_apps

### DISCUSSION THREAD

**Discussion by alanjhewitt** [MetOffice/simulation-systems#240] [LFRic]
there is a page that describes some of these in the UM
https://code.metoffice.gov.uk/trac/um/wiki/KnownUMFailurePoints

**Comment by iboutle** [comment]
No, but it's a good idea. Is there a specific reason you ask?

Relative to what is listed there:
- we don't have packing so the top one isn't an issue
- NaN generation in physics usually presents as a segfault in screen_tq
- the bottom 2 are usually trapped by helpful error messages in the transport scheme

**Reply by alanjhewitt** [reply]
the known failure points can be a bit of a wild goose chase of debugging when something is wrong, because the error is usually unrelated to the error message.

wayne is trying to run a suite with chemistry/aerosols and it gets up to timestep six and crashes in advection with a value outside of reasonable bounds. we figure it is more likely to be a UKCA problem that didnt crash at that point, so having a list of known failure points would be handy to compare with where it crashes.
